master/static/srv/enrich_task_logs.py

```diff
@@ -130,13 +136,16 @@ def ship(self) -> None:
         tries = 0
         while tries < max_tries:
             try:
-                api.post(self.master_url, "task-logs", self.logs)
+                api.post(self.master_url, "task-logs", self.logs, cert=self.cert)
                 self.logs = []
                 return
-            except errors.APIException as e:
+            except Exception as e:
```

Comment:
don't do `as e` if `e` is not being used.

Reply:
e is being used, it's being printed in ` print(
                        "failed to ship logs: {e}\nLogs to Ship: {ln(self.logs)}}",

Reply:
doh